kc-cloudtrust-testsuite/kc-cloudtrust-db-access/src/main/java/io/cloudtrust/keycloak/test/dbprovider/DatabaseRealmResourceProvider.java

```diff
@@ -0,0 +1,81 @@
+package io.cloudtrust.keycloak.test.dbprovider;
+
+import jakarta.persistence.EntityManager;
+import jakarta.ws.rs.Consumes;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.POST;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.PathParam;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.core.MediaType;
+import jakarta.ws.rs.core.Response;
+import org.jboss.logging.Logger;
+import org.keycloak.connections.jpa.JpaConnectionProvider;
+import org.keycloak.models.KeycloakSession;
+import org.keycloak.models.UserModel;
+import org.keycloak.services.resource.RealmResourceProvider;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class DatabaseRealmResourceProvider implements RealmResourceProvider {
+    private static final Logger LOG = Logger.getLogger(DatabaseRealmResourceProvider.class);
+
+    private KeycloakSession session;
+
+    public DatabaseRealmResourceProvider(KeycloakSession session) {
+        this.session = session;
+        LOG.debugf("Instanciated DatabaseRealmResourceProvider for request %s %s",
+                session.getContext().getHttpRequest().getHttpMethod(),
+                session.getContext().getHttpRequest().getUri().getPath());
+    }
+
+    @Override
+    public void close() {
+        // Nothing to do
+    }
+
+    @Override
+    public Object getResource() {
+        return this;
+    }
+
+    @GET
+    @Path("hello")
```

Comment:
path hello ? is it really something we need ?

Reply:
Just a simple GET API method to check that this TEST module is working correctly.
This new module has only been added for tests purpose.